src/extension/popup.js

```diff
@@ -220,6 +220,7 @@ function populateCharacter(response) {
         }
         if (response["actions"].includes("Channel Divinity: Sacred Weapon")) {
             e = createHTMLOption("paladin-sacred-weapon", false, character_settings);
+            options.append(e);
```

Comment:
oups :D

Reply:
Yeah, it was the best, I was like "Y u no test properly!" And then I found it...